---a/net/minecraft/network/play/server/S0BPacketAnimation.java
+++b/net/minecraft/network/play/server/S0BPacketAnimation.java
@@ -1,48 +1,44 @@
 package net.minecraft.network.play.server;
 
 import java.io.IOException;
+
 import net.minecraft.entity.Entity;
 import net.minecraft.network.INetHandler;
 import net.minecraft.network.Packet;
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class S0BPacketAnimation implements Packet
-{
+
+public class S0BPacketAnimation implements Packet {
+
     private int entityId;
     private int type;
     private static final String __OBFID = "CL_00001282";
 
-    public S0BPacketAnimation() {}
+    public S0BPacketAnimation() {
+    }
 
-    public S0BPacketAnimation(Entity ent, int animationType)
-    {
+    public S0BPacketAnimation(Entity ent, int animationType) {
         this.entityId = ent.getEntityId();
         this.type = animationType;
     }
 
-    public void readPacketData(PacketBuffer buf) throws IOException
-    {
+    public void readPacketData(PacketBuffer buf) throws IOException {
         this.entityId = buf.readVarIntFromBuffer();
         this.type = buf.readUnsignedByte();
     }
 
-    public void writePacketData(PacketBuffer buf) throws IOException
-    {
+    public void writePacketData(PacketBuffer buf) throws IOException {
         buf.writeVarIntToBuffer(this.entityId);
         buf.writeByte(this.type);
     }
 
-    public void func_180723_a(INetHandlerPlayClient p_180723_1_)
-    {
+    public void func_180723_a(INetHandlerPlayClient p_180723_1_) {
         p_180723_1_.handleAnimation(this);
     }
 
-    @SideOnly(Side.CLIENT)
-    public int func_148978_c()
-    {
-        return this.entityId;
+    public void processPacket(INetHandler handler) {
+        this.func_180723_a((INetHandlerPlayClient) handler);
     }
 
-    public void processPacket(INetHandler handler)
-    {
-        this.func_180723_a((INetHandlerPlayClient)handler);
-    }
-
-    @SideOnly(Side.CLIENT)
-    public int func_148977_d()
-    {
-        return this.type;
-    }
-}
+}
